packages/sdks/typescript/src/utils/types.ts

```diff
@@ -0,0 +1,32 @@
+export type CommitBaseParams = {
+  projectId: string
+  commitUuid: string
+}
+export type GetDocumentParams = {
+  projectId: string
+  commitUuid: string
+  documentPath: string
+}
+
+export enum HandlerType {
+  RunDocument = 'run-document',
+  GetDocument = 'get-document',
```

Comment:
What's the use case for get document?

Reply:
If we open source the compiler this is useful so users can get the document, compile on their end and do whatever they want with it. Otherwise it's pretty useless.